Makefile

```
@@ -0,0 +1,13 @@
+dind/% operator/% runner/%:
+	$(MAKE) -C $(@D) $(@F)
+
+upgrade:
+	$(eval LATEST := $(shell curl -s https://api.github.com/repos/actions/runner/releases/latest | jq -r '.tag_name' | sed -E 's/^v//'))
```

You can use jq's left-trim-string function:
```suggestion
	$(eval LATEST := $(shell curl -s https://api.github.com/repos/actions/runner/releases/latest | jq -r '.tag_name | ltrimstr("v")'))
```

---

done